WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainCoordinator.swift

```diff
@@ -36,11 +41,13 @@ class RegisterDomainCoordinator {
     init(site: Blog?,
          domainPurchasedCallback: RegisterDomainCoordinator.DomainPurchasedCallback? = nil,
          analyticsSource: String = "domains_register",
-         crashLogger: CrashLogging = .main) {
+         crashLogger: CrashLogging = .main,
+         domainRegistrationService: RegisterDomainDetailsServiceProxyProtocol = RegisterDomainDetailsServiceProxy()) {
         self.site = site
         self.domainPurchasedCallback = domainPurchasedCallback
         self.crashLogger = crashLogger
         self.analyticsSource = analyticsSource
+        self.domainRegistrationService = domainRegistrationService
```

Comment:
This service was instantiated within the `createCart` method, but I've moved to `init` for better dependency injection, allowing easier unit testing.